erpnext/accounts/report/profit_and_loss_statement/profit_and_loss_statement.py

```diff
@@ -32,7 +32,7 @@ def execute(filters=None):
 
 	chart = get_chart_data(filters, columns, income, expense, net_profit_loss)
 
-	default_currency = frappe.get_cached_value('Company', filters.company, "default_currency")
+	default_currency = filters.presentation_currency or frappe.get_cached_value('Company', filters.company, "default_currency")
```

Comment:
maybe rename the variable to `currency`:
```suggestion
	currency = filters.presentation_currency or frappe.get_cached_value('Company', filters.company, "default_currency")
```